java/ingest/ingest-runner/src/test/java/sleeper/ingest/IngestRecordsIT.java

```diff
@@ -78,18 +80,10 @@
 class IngestRecordsIT extends IngestRecordsTestBase {
```

Comment:
There are some tests that haven't been done yet:

shouldWriteRecordsSplitByPartitionByteArrayKey
shouldWriteRecordsSplitByPartition2DimensionalByteArrayKey
shouldWriteRecordsSplitByPartition2DimensionalDifferentTypeKeysWhenSplitOnDim1